o/opticall/opticall-0.8.1-foss-2022a.eb

```diff
@@ -0,0 +1,28 @@
+name = 'opticall'
+version = '0.8.1'
+
+homepage = 'https://github.com/molgenis/molgenis-pipelines'
+description = """This distribution contains several pipelines/protocols/parameter files,
+which you can use 'out-of-the-box' to align and impute your NGS data using MOLGENIS Compute."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
```

Comment:
Documentation is poor, but at first glance I do not see dependencies on OpenBLAS, LAPACK, FFTW, etc. So, did you already try to build this with the GCCcore part of foss? If opticall does not need all the extra libraries and the GCC compilers are sufficient, then it is better not to include unnecessary deps.